includes/wc-core-functions.php

```diff
@@ -158,7 +158,13 @@ function wc_update_order( $args ) {
  */
 function wc_get_template_part( $slug, $name = '' ) {
 	$cache_key = sanitize_key( implode( '-', array( 'template-part', $slug, $name ) ) );
-	$template  = (string) wp_cache_get( $cache_key, 'woocommerce' );
+	$template  = wp_cache_get( $cache_key, 'woocommerce' );
```

Comment:
Just wondering if it wouldn't be easier to make the version part of the cache key? Then you wouldn't need to change the rest and if the caching depends on the version anyway, then it perhaps should be part of cache key?

Reply:
A good suggestion, will make the changes.